src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -68,6 +69,7 @@ public static void GetDirectoryName_SpaceThrowOnWindows_Core()
             if (PlatformDetection.IsWindows)
             {
                 AssertExtensions.Throws<ArgumentException>("path", null, () => Path.GetDirectoryName(path));
+                Assert.Equal(string.Empty, new string(Path.GetDirectoryName(new ReadOnlySpan<char>())));
```

Comment:
This is not equivalent to the string test, which is using a space, not empty string. Take care..